COPYING

```diff
@@ -2,7 +2,7 @@ Unless specified otherwise in individual files or directories, the
 contents of this package is covered by the following copyright, license,
 and disclaimer:
 
-Copyright 2006 - 2015 Tarsnap Backup Inc.
+Copyright 2006 - 2016 Tarsnap Backup Inc.
 All rights reserved.
```

Comment:
Split the patch and corrected it here:
https://github.com/Tarsnap/tarsnap/pull/117